cif_pow.dic

```diff
@@ -7924,16 +8033,61 @@ save_pd_instr.soller_eq_spec_detc
 
 save_
 
+save_pd_instr_detector.diffrn_detector_id
+
+    _definition.id                '_pd_instr_detector.diffrn_detector_id'
+    _definition.update            2025-06-26
+    _description.text
+;
+    The diffrn detector (see _diffrn_detector.id) to which the pdCIF
+    detector belongs.
+;
+    _name.category_id             pd_instr_detector
+    _name.object_id               diffrn_detector_id
+    _name.linked_item_id          '_diffrn_detector.id'
+    _type.purpose                 Link
+    _type.source                  Related
+    _type.container               Single
+    _type.contents                Text
+
+save_
+
+save_pd_instr_detector.diffrn_id
+
+    _definition.id                '_pd_instr_detector.diffrn_id'
+    _definition.update            2025-06-30
+    _description.text
+;
+    A diffrn id (see _diffrn.id) associated with the pdCIF detector.
+
+    Included to maintain compatibility with DIFFRN_DETECTOR and imgCIF.
+    Do not use this value. Use the value associated with the appropriate
+    _pd_diffractogram.diffrn_id in any lookup associated with DIFFRN_DETECTOR.
+;
+    _name.category_id             pd_instr_detector
+    _name.object_id               diffrn_id
+    _name.linked_item_id          '_diffrn.id'
+    _type.purpose                 Link
+    _type.source                  Related
+    _type.container               Single
+    _type.contents                Word
+
+save_
+
 save_pd_instr_detector.id
 
     _definition.id                '_pd_instr_detector.id'
-    _definition.update            2023-06-09
+    _definition.update            2025-06-26
     _description.text
 ;
-    A code which identifies the detector or channel number in a
-    position-sensitive, energy-dispersive or other multiple-detector
-    instrument for which the individual instrument geometry is being
-    defined. Where a single detector is used, this may be omitted.
+    An arbitrary code which identifies the detector.
+
+    Although it is possible, it should not be used to describe a channel
```

Comment:
I would delete "Although it is possible" because that is opening a gate for software authors. Perhaps the text of Vol G might discuss legacy use as a channel identifier. I wonder how many datasets are around that actually use this as a channel identifier in a way that is problematic?